jobs/update-legal-filings/pyproject.toml

```diff
@@ -0,0 +1,50 @@
+[tool.poetry]
+name = "update-legal-filings"
+version = "0.1.0"
+description = "Update legal filings job"
+authors = ["Hongjing Chen <Hongjing.Chen@gov.bc.ca>"]
+license = "BSD-3-Clause"
+
+[tool.poetry.dependencies]
+python = "^3.10"
+Flask-Moment = "^1.0.5"
+Flask-Script = "^2.0.6"
+Flask = "^2.3.3"
+Jinja2 = "^3.1.2"
+MarkupSafe = "^2.1.3"
+Werkzeug = "^2.3.7"
+aniso8601 = "^9.0.1"
+attrs = "^23.1.0"
+blinker = "^1.6.2"
+certifi = "^2023.7.22"
+click = "^8.1.7"
+ecdsa = "^0.18.0"
+flask-jwt-oidc = "^0.3.0"
+gunicorn = "^20.1.0"
+itsdangerous = "^2.1.2"
+jsonschema = "^4.19.0"
+pyasn1 = "^0.5.0"
+pyrsistent = "^0.19.3"
+python-dotenv = "^1.0.0"
+python-jose = "^3.3.0"
+pytz = "^2023.3.post1"
+rsa = "^4.9"
+sentry-sdk = "^1.31.0"
+six = "^1.16.0"
+urllib3 = "^1.22.0"
+google-auth = "^2.21.0"
+google-cloud-pubsub = "^2.17.1"
+colin_api = { git = "https://github.com/bcgov/lear.git", subdirectory = "colin-api", branch = "dev_legal_name_changes" }
+legal_api = { git = "https://github.com/bcgov/lear.git", subdirectory = "legal-api", branch = "dev_legal_name_changes" }
+registry-schemas = {git = "https://github.com/bcgov/business-schemas.git", branch = "dev_legal_name_changes"}
+simple_cloudevent = { git = "https://github.com/daxiom/simple-cloudevent.py" }
```

Comment:
Can we move `simple_cloudevent = { git = "https://github.com/daxiom/simple-cloudevent.py" }` above `colin_api = { git = "https://github.com/bcgov/lear.git", subdirectory = "colin-api", branch = "dev_legal_name_changes" }`.

We typically have the registry modules listed last.